mathesar/api/viewsets/tables.py

```diff
@@ -105,18 +107,21 @@ def previews(self, request, pk=None):
             raise ValidationError("Incorrect number of columns in request.")
 
         table_data = TableSerializer(table, context={"request": request}).data
-        try:
+        with exception_transformer({**dict.fromkeys([DataError, IntegrityError], ExceptionTransformerDetail(400,
```

Comment:
@silentninja This function call for `exception_transformer` is a little hard to follow. It would be clearer if we could declare the dictionaries in a separate line and pass them here.

Reply:
I also think it would be clearer to instantiate the named tuple using keyword arguments (e.g. `status_code=400`, etc.) so that we don't have to remember the order in which to pass arguments and it's clearer what's going on without having to look at the original function signature.